glean_parser/kotlin.py

```diff
@@ -163,9 +227,11 @@ def output_kotlin(objs, output_dir, options={}):
                     extra_args=extra_args,
                     namespace=namespace,
                     has_labeled_metrics=has_labeled_metrics,
-                    has_gecko_datapoints=has_gecko_datapoints,
                     glean_namespace=glean_namespace,
                 )
             )
             # Jinja2 squashes the final newline, so we explicitly add it
             fd.write('\n')
+
+    # TODO: Maybe this should just be a separate outputter?
```

Comment:
Note: creating a custom outputter would require duplicating some of the kotlin code anyway. It would also need changes on a-c to call in the correct outputter when in engine-gecko-*